server/src/logic.js

```diff
@@ -6,33 +6,27 @@ import { JWT_SECRET } from './config';
 import { schedulePerms, eventPerms } from './perms';
 
 // reusable function to check for a user with context
-const getAuthenticatedUser = ctx => ctx.user.then((user) => {
-  // null means we couldn't find the user record
-  if (!user) {
-    // eslint-disable-next-line prefer-promise-reject-errors
-    return Promise.reject('Unauthorized: Invalid user ID');
-  }
-  return user;
-});
+const getAuthenticatedUser = ctx =>
```

Comment:
again with the whitespace changes making it really hard to review.. if you're going to do this stuff I'd really appreciate putting it in a separate commit so I can review it separately.

Reply:
ok I got about halfway through and gave up. Can you comment on the changes?

Reply:
whitespaces? I have some auto-formatting in vscode if that's what is causing the issue

Reply:
this whole file has changed so much I couldn't be bothered to pick out the actual (non-whitespace) changes. Can you highlight them with inline comments?